pkg/webhook/controlplane/add.go

```diff
@@ -20,22 +26,43 @@ import (
 )
 
 var (
-	logger = log.Log.WithName("openstack-controlplane-webhook")
+	logger                           = log.Log.WithName("openstack-controlplane-webhook")
+	versionConstraintGreaterEqual198 *semver.Constraints
```

Comment:
We could add a TODO to clean up the gardener version check in 3 months for example.

```suggestion
	// TODO(LucaBernstein): Clean up the gardener version check after October/2024.
	versionConstraintGreaterEqual198 *semver.ConstraintsversionConstraintGreaterEqual198 *semver.Constraints
```